CryptoBook/validators.py

```diff
@@ -1,38 +1,56 @@
 from cerberus import Validator, DocumentError
 from datetime import datetime
-
+import ccxt.async_support as ccxt_async
```

Comment:
Good idea pulling out validators into their own file.

Reply:
Makes more sense, I agree. However, I believe a new codebase style should be in order. As of right now, the following is occurring:
```
API -> Validator (function return) -> API -> Utils (function return) -> API (function return) -> Client
```
I believe perhaps a better format would be:
```
API -> Validator (calls) -> Utils (function return) -> Validator (function return) -> API (function return) -> Client
```
This is due to the fact some errors might be thrown by the `utils.py` file that I don't know if it would be worth catching it inside `utils`. Stuff such as 'max tries has been reached`, `DDOS protection`, etc. This way utils is responsible solely to pulling data, and the verifier is the middle-man between the API and the pulling of data with `try` block ready to pick-up _specific_ exceptions (`ccxt.ddos.exception`, `ccxt.maxtries.exception`) and the validator would be ready to return to the API either the data or error. 

Opening up an Issue on this topic for further discussion!

Reply:
Moving conversation to #22 .